src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public class ColorSupplier {private Random random = new Random();
```

Comment:
Code style issue: Please add an empty line between class declaration and the first method or field.